internal/service/ipam/model_ipamsvc_subnet.go

```diff
@@ -456,8 +456,8 @@ func (m *IpamsvcSubnetModel) Flatten(ctx context.Context, from *ipam.IpamsvcSubn
 	}
 	m.Address = flex.FlattenStringPointer(from.Address)
 	m.AsmConfig = FlattenIpamsvcASMConfig(ctx, from.AsmConfig, diags)
-	m.AsmScopeFlag = flex.FlattenInt64(int64(*from.AsmScopeFlag))
-	m.Cidr = flex.FlattenInt64(int64(*from.Cidr))
+	m.AsmScopeFlag = flex.FlattenInt64(*from.AsmScopeFlag)
```

Comment:
should be like this .. these were generated with old code so didn't get updated 
```suggestion
	m.AsmScopeFlag = flex.FlattenInt64Pointer(from.AsmScopeFlag)
```

Reply:
Updated

Reply:
same comment applies for cidr, RebindTime and RenewTime

Reply:
same comment applies for cidr, RebindTime and RenewTime